src/components/kytos/table/Table.vue

```diff
@@ -95,4 +197,27 @@ export default {
       color: $fill-text-h
       background-color: rgba(0, 0, 0, 0.15)
 
+.next-previous
+  display: flex
+  justify-content: flex-end
+  align-items: center
+
+  button
+    background: inherit !important
+    svg
+      width: 14px
+      height: 14px
+
+  .page-number
+    display: inline-block
+    margin: 2px
+    padding: 2px 5px
+    font-weight: bold
+    font-family: Tahoma, serif
```

Comment:
Tahoma is not listed anywhere else in the UI project, right?

Reply:
Well, I used a button as example, and when I inspected it the font showed was Tahoma... But yes, I don't think that this is listed anywhere else